sire/sire.py

```diff
@@ -313,6 +330,12 @@ def sire(name, mkdocs=True, virtualenv=True, git=True, exclude=None, interactive
     if git:
         subprocess.call(f"git init {name}".split())
         paths.update({".gitignore", ".pre-commit-config.yaml"})
+        github_username = _obtain_github_username()
```

Comment:
I believe you just need to change this to `formatters['github_username'] = _obtain_github_username(), then delete the conditional and below, as that printing will happen later.